### ISSUE TITLE
NTR: Pancreatic biopsy: intraductal papillary mucinous tumors (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9296]
New term request Pancreatic biopsy: intraductal papillary mucinous tumors (elthat_130221205353): IPMN  UMLS: 4511687

**Comment by pnrobinson** [comment]
@MickeySegal
There are many subtypes of this kind of cancer -- https://www.ebi.ac.uk/ols/ontologies/ncit/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FNCIT_C38342
Should we name this term
Pancreatic intraductal papillary-mucinous neoplasm and xref to the NCIT or is there a more specific type ?

**Comment by pnrobinson** [comment]
@MickeySegal thoughts on the above?

**Comment by MickeySegal** [comment]
https://www.ncbi.nlm.nih.gov/books/NBK274564/ argues for the less granular approach: "Intraductal papillary mucinous neoplasms (IPMN) have been reported, which may present with variable grades of dysplasia"

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6521861/ seems to take a less granular approach as well.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000657
- label: Pancreatic intraductal papillary mucinous neoplasm
- parents: [Neoplasm of the pancreas (HP:0002894)](https://hpo.jax.org/app/browse/term/HP:0002894)
- pmid: PMID:35070031
- def: Intraductal papillary mucinous neoplasm (IPMN) is an exocrine neoplasm of the pancreas consisting of epithelial cells growing within the pancreatic ducts (main pancreatic duct or its major branches) and producing mucin. IPMN is a mucin-producing pancreatic cystic tumor. IPMN contains epithelial cells that can create papillary projections.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9296
- synonyms:
